sdcm/nemesis.py

```diff
@@ -1775,29 +1775,31 @@ def disrupt_mgmt_backup(self):
         self._mgmt_backup(backup_specific_tables=False)
 
     def _mgmt_backup(self, backup_specific_tables):
-        # TODO: When cloud backup is supported - add this to the below 'if' statement:
-        #  and not self.cluster.params.get('use_cloud_manager')
-        if not self.cluster.params.get('use_mgmt'):
-            raise UnsupportedNemesis('Scylla-manager configuration is not defined!')
-        if not self.cluster.params.get('backup_bucket_location'):
-            raise UnsupportedNemesis('backup bucket location configuration is not defined!')
-
         mgr_cluster = self.cluster.get_cluster_manager()
```

Comment:
this is no good, as if you call `self.cluster.get_cluster_manager()` with `use_mgmt=False` (the case your next if does), it will fail, and the `use_cloud_manager` won't be able to work, so you should also extend in the `cluster.py` method the option to run if `use_cloud_manager=True`

Reply:
thanks, just added the first "if" statement as was originally mentioned in the comment.. fixed.